authorization/index.md

```diff
@@ -124,11 +124,20 @@ On receiving the launch notification, the app would query the issuer's
     Accept: application/json
 
 The metadata response contains (among other details) the EHR's authorization
-URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.
+URL for use below. For details about how the EHR publishes the relevant OAuth URLs, <a href="{{site.baseurl}}authorization/conformance-statement">see here</a>.  
+Later, when the app prepares a list of access scopes to request from the EHR authorization server, it will bind to the existing EHR context by including the launch notification in the scope.
+
+#### Standalone launch sequence
+If the app is a standalone app that launches from outside the EHR, it will launch from its registered URL without a launch id.  Such an app will declare its launch context requirements by adding specific scopes to the authorization request.  The EHR’s "authorize" endpoint will acquire the context the app needs and making it available.   
+
+For example:
+If the app needs patient context, the EHR AS may provide the end-user with a patient selection widget.  For full details, see SMART launch context parameters (/authorization/scopes-and-launch-context).
+
+*	launch/patient - to indicate that the app needs to know a patient ID
```

Comment:
When referring to fixed coded values, use back-quotes to ensure appropriate rendering.

In other words, write **markdown like**:

	`launch/patient` - to indicate that the app needs to know a patient ID

To cause it to **render as**:

`launch/patient` - to indicate that the app needs to know a patient ID